src/braket/experimental/autoqasm/doc/decorators.md

```diff
@@ -87,8 +85,6 @@ def ch(q0: aq.Qubit, q1: aq.Qubit):
 def main():
```

Comment:
nit: The main program here is called `main`. In the gate_calibration section below, it's called `my_program`. Would be good to make them consistent.